tests/js/common/shell/shell-index.js

```diff
@@ -1198,15 +1198,29 @@ function parallelIndexSuite() {
 
     testCreateInParallel: function () {
       let noIndices = 80;
-      if (platform.substr(0, 3) === 'win') {
-        // Relax condition for windows - TODO: fix this.
-        noIndices = 40;
-      }
+      let contextData = {
+        before: require("internal").serverStatistics().v8Context
+      };
+
+      // Warmup V8-Contexts first:
       for (let i = 0; i < noIndices; ++i) {
-        let command = 'require("internal").db._collection("' + cn + '").ensureIndex({ type: "hash", fields: ["value' + i + '"] });';
+        let command = 'require("internal").sleep(0.5);';
```

Comment:
I am very reluctant to accept this change.
This creates 40 (or 80 on !Windows) tasks that sleep half a second. This will not help anything but in the worst case delay test execution considerably.
Please let's stop with such things.

Reply:
it spawns .js contexts which otherwise would happen in the lines below. Spawning new contexts doesn't come for free, and influences the test behaviour.
Probably an API-Call to warm up N-contexts would be the better solution for this.

Reply:
Why...? Contexts are spawned automatically when tasks are dispatched.
Any warmup etc. is just unnecessary bloat.
If the contexts take too long to be spawned, we should address this. But a warmup routine will not fix it.